FrontEnd/src/components/profileList/ProfileCard.jsx

```diff
@@ -89,12 +112,18 @@ export default function ProfileCard({ isAuthorized, data }) {
           </div>
         </div>
       </Link>
-      <StarForLike
-        isSaved={isSaved}
-        isAuthorized={isAuthorized}
-        ownProfile={ownProfile}
-        handleClick={isSaved ? handleDeleteSaved : handleSave}
-      ></StarForLike>
+      <div className={css['bell-container']}>
+        <BellForUpdates
+          className={savedIsUpdated ? '' : 'hidden'}
+          savedIsUpdated={savedIsUpdated}
```

Comment:
I'm wondering why we can't use `data.saved_is_updated` directly.

Reply:
Fixed it